src/App.js

```diff
@@ -112,7 +112,7 @@ const App = () => {
     const url = `${baseURL}/items/${id}.json`;
     const requestOptions = { method: 'DELETE', headers };
 
-    // Reset alerts in case user did not dismiss them
+    // Reset alerts in case user did not dismiss them.  Still necessary?
     setExpenseAdded(undefined);
     setExpenseDeleted(undefined);
```

Comment:
See comment above for lines 85 - 87 of `App.js`

Reply:
same as above